udf-server-python/src/io/aklivity/zillabase/udf-server.py

```diff
@@ -33,6 +36,43 @@ def fetch_classes():
                         class_names.append(file)
     return class_names
 
+def install_package(package_name, version=None):
+    package_str = f"{package_name}=={version}" if version else package_name
+    with open(os.devnull, 'w') as devnull:
+        subprocess.check_call(
+            [sys.executable, "-m", "pip", "install", package_str],
+            stdout=devnull,
+            stderr=devnull
```

Comment:
Doing silent installs and not printing the output seems like a good idea, but if the install fails, then we should do what to see that there is a problem. Will this block printing any errors with an install?